### PR TITLE
LRAC-8130 Add ViewAllWebContentShownInAssetList and CanNavigateToWebContentOverviewPage testcase

### PR DESCRIPTION
**Pull request by ViniLopes20**

### CODE REVIEW

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/AssetsBlogs.testcase [hunk 1]
```diff
@@ -115,10 +115,9 @@ definition {
 
 		ACAssets.accessAssets(titleName = "Blogs AC Title");
 
-		AssertTextEquals(
-			locator1 = "ACAssets#MATRIC_VALUE",
-			matricTitle = "Views",
-			value1 = "4");
+		ACAssets.viewVisitorsBehaviorMetric(
```
Comment (alexcandidos) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/AssetsBlogs.testcase:118]: Awesome @ViniLopes20 !! 👏🏼

File: portal-web/test/functional/com/liferay/portalweb/macros/analyticscloud/ACDXPSettings.macro [hunk 2]
```diff
@@ -127,4 +127,18 @@ definition {
 		Click(locator1 = "ACDXPSettings#AC_GENERIC_ADD_OR_CONFIRMATION_BUTTON");
 	}
 
+	macro addUser {
```
Comment (alexcandidos) [inline portal-web/test/functional/com/liferay/portalweb/macros/analyticscloud/ACDXPSettings.macro]: hey @ViniLopes20 , why are we encapsulating `JSONUser.addUser` and `User.logoutAndLoginPG`under `ACDXPSettings#addUser` and `ACDXPSettings#userLogin` respectively?

Just to keep them under our macros? 🤔

Comment (ViniLopes20) [reply portal-web/test/functional/com/liferay/portalweb/macros/analyticscloud/ACDXPSettings.macro]: Is a good point @alexcandidos, but I thought the test would be less polluted if they were in our macros. But thinking about it now I don't think that much changes, what do you think?

**Comment by ViniLopes20** [comment]
It's covered by https://github.com/alexcandidos/liferay-portal/pull/469

**Pull request closed without merging**